spec/money_column_spec.rb

```diff
@@ -8,7 +8,8 @@ class MoneyRecord < ActiveRecord::Base
 
   money_column :price
   money_column :prix, currency_column: :devise
-  money_column :price_usd, currency_column: false, currency: 'USD'
+  money_column :price_usd, currency: 'USD'
+  money_column :price4, currency_read_only: true
```

Comment:
Instead of adding another column, WDYT about using another AR class like the `MoneyWithValidation` specs?